tests/queries/0_stateless/03261_secondary_indicies_by_default.sql

```diff
@@ -0,0 +1,85 @@
+set mutations_sync = 2;
+
+DROP TABLE IF EXISTS tbl;
+DROP TABLE IF EXISTS tbl2;
+
+CREATE TABLE tbl
+(
+    key Int,
+    x Int,
+    y Int,
+    INDEX x_idx x TYPE minmax GRANULARITY 1
+)
+Engine=MergeTree()
+ORDER BY key;
+
+SELECT 'Default';
+SHOW INDICES FROM tbl;
```

Comment:
Instead of `SHOW INDICES` you perhaps like to check system.data_skipping_indices ([docs](https://clickhouse.com/docs/en/operations/system-tables/data_skipping_indices)). It shows specifically only skipping indices (no PK) and the output is less cluttered.

Reply:
addressed it